source/integration_tools.html.md

```diff
@@ -0,0 +1,73 @@
+# Integration Tools
```

Comment:
Can we hold the changes here until after the bug bash and we confirm that we are not blocked for launch?

Reply:
It doesn't link to the tool so i think we can release the docs and then have the tool come later